burrow/README.md

```diff
@@ -0,0 +1,67 @@
+
+## Prometheus
+
+Burrow supports prometheus exporter.
+https://github.com/linkedin/Burrow/blob/master/core/internal/httpserver/prometheus.go
+
+
+If you use datadog and promehteus exporter, you need `podAnnotation` parameter for metrics.
+https://docs.datadoghq.com/integrations/prometheus/
+
+
+## Configuration
+
+The following table lists the configurable parameters of the Burrow chart and their default values.
+
+| Parameter                | Description             | Default        |
+| ------------------------ | ----------------------- | -------------- |
+| `replicaCount` | replica count | `1` |
+| `image.repository` | image repository | `"docker.pkg.github.com/linkedin/burrow/burrow"` |
+| `image.tag` | image tag | `"v1.3.4"` |
+| `image.pullPolicy` | image pull policy | `"IfNotPresent"` |
+| `imagePullSecrets` | image pull secrets  | `[]` |
+| `serviceAccount.create` | if true, a service account will be created  | `true` |
+| `serviceAccount.annotations` | service account annotations | `{}` |
+| `serviceAccount.name` | existing ServiceAccount to use (ignored if serviceAccount.create=true) | `""` |
+| `podAnnotations` | pod annotations | `{}` |
+| `podSecurityContext` | pod security context | `{}` |
+| `securityContext` | security context | `{}` |
+| `resources` | pod resources  | `{}` |
+| `nodeSelector` | node selector | `{}` |
+| `tolerations` | tolerations | `[]` |
+| `affinity` | affinity | `{}` |
+| `service.type` | service type | `"ClusterIP"` |
+| `service.port` | service port  | `8080` |
+| `readinessProbe.enable` | if true, readinessProbe enable | `true` |
+| `readinessProbe.periodSeconds` | readinessProbe period seconds | `60` |
+| `readinessProbe.timeoutSeconds` | readinessProbe timeout seconds | `3` |
+| `readinessProbe.initialDelaySeconds` | readinessProbe initial delay seconds | `10` |
+| `livenessProbe.enable` | if true, livenessPribe enable | `true` |
+| `livenessProbe.periodSeconds` | livenssProbe period seconds  | `60` |
+| `livenessProbe.timeoutSeconds` | livenessProbe timeout seconds | `1` |
+| `livenessProbe.initialDelaySeconds` | livessProbe initial delay seconds | `30` |
+| `config.burrow.toml` | burrow [config](https://github.com/linkedin/Burrow/wiki/Configuration) | `---` |
+| `datadog.enabled` | if true, datadog works as sidecar, | `false` |
+| `datadog.apiKey` | datadog apikey  | `""` |
+| `datadog.logLevel` | datadog log level | `"INFO"` |
+| `datadog.image.repository` | datadog image repository | `"datadog/agent"` |
+| `datadog.image.tag` | datadog image tag | `7` |
+| `datadog.image.pullPolicy` | datadog image pull policy | `"Always"` |
+| `datadog.image.resources` | datadog image resources | `null` |
+| `datadog.checksd.clusters` | datadog checksd parameter | `["local"]` |
+| `datadog.checksd.tags` | datadog checksd metrics tags | `null` |
+| `datadog.checksd.burrowCheckScript` | if you change datadog checksd script for burrow, you can use your script | `null` |
```

Comment:
@cw-sakamoto 
Do you need Datadog?
There is auto dIscovery for kubernetes in Datadog.
Why do you want to run it in sidecar?
https://docs.datadoghq.com/ja/agent/kubernetes/integrations/?tab=kubernetes